src/main/java/frc/robot/subsystems/drive/SwerveModule.java

```diff
@@ -157,6 +171,18 @@ public void burnFlashSparks() {
     turningSparkMax.burnFlash();
   }
 
+  public void considerZeroingEncoder() {
+    if (Math.abs(turningAbsoluteEncoder.getPosition()) < 0.01) {
+      return;
+    }
+    if (Math.abs(getEncoderRelativePositionRad() - getEncoderAbsPositionRad())
+        > DriveCal.TURNING_ENCODER_ZEROING_THRESHOLD_DEG) {
+      turningRelativeEncoder.setPosition(getEncoderAbsPositionRad() + chassisAngularOffsetRadians);
+      // turningPIDController.reset(getEncoderRelativePositionRad());
```

Comment:
I think this should be minus?? Also set pid controller setpoint here to the current position?

Reply:
Done